### PR TITLE
Create sqlcmd action

### PR DESCRIPTION
**Pull request by koiralaprt**
Github Action for running sql query files:
- Installs sqlcmd tool in the github runner
- takes in db_user, db_password, query file path, db_host url as inputs 
- executes query as configured from the inputs, writes the output/results in a text file with the SHA as the filename

### CODE REVIEW

File: sqlcmd/action.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        curl https://packages.microsoft.com/config/ubuntu/20.04/prod.list | sudo tee /etc/apt/sources.list.d/msprod.list
+        sudo apt-get update 
+        sudo apt-get install mssql-tools unixodbc-dev
+        export PATH=$PATH:/opt/mssql-tools/bin
+        echo "Sqlcmd installation complete"
+      shell: bash
+    - run: |
+        echo "Running the query file ${{ inputs.query_file }}"
+        GIT_HASH="$GITHUB_SHA"
+        sqlcmd -S ${{ inputs.db_host }},${{ inputs.db_port }} -U ${{ inputs.db_user }} -P ${{ inputs.db_password }} -b -i ${{ inputs.query_file }} -o "${GIT_HASH}.txt"
```
Comment (silwalanish) [inline sqlcmd/action.yml:36]: Won't this print the DB credentials to the github-action logs?

Comment (koiralaprt) [reply sqlcmd/action.yml:36]: Don't think so. The inputs come through github secrets thus shouldn't be able to be seen in the shell through the logs.

Comment (mesaugat) [inline sqlcmd/action.yml:36]: @koiralaprt Why are we writing the output to a text file? Isn't the whole CI job ephemeral in nature? Once all the steps are done, we won't be able to reach the machine that the process ran on? This means the output file won't exist after the action termination? Why not just print the output to stdout?

Comment (koiralaprt) [reply sqlcmd/action.yml:36]: Stdout seems to reflect the output without the proper details. eg '1 rows affected' where the user cant actually see what was affected. tried "select * from some_table" for this. The output written into a file can be thus copied somewhere, to s3 (as I've planned) in the next step so theres a way the we can actually see the outputs and **have a history** too. This is implemented here for reference-> https://github.com/laudio/etl_sql_manual_queries/pull/1/files#:~:text=%2D%20name%3A%20Copy%20output,region%20us%2Deast%2D1

Comment (koiralaprt) [reply sqlcmd/action.yml:36]: @mesaugat 
🆙

**Review by aayushsthapit — APPROVED** [review]

**Review by samirsilwal — APPROVED** [review]

**Review by unN00b — APPROVED** [review]

**Pull request merged**